javascript-functions/main.js

```diff
@@ -0,0 +1,35 @@
+function convertMinutesToSeconds(minutes, seconds) {
+  var multiply = minutes * seconds;
+  return multiply;
+}
+
+var convertMinutesToSecondsResult = convertMinutesToSeconds(5, 60);
+console.log('convertMinutesToSecondsResult:', convertMinutesToSecondsResult);
+
+function greet(Hey, Beavis) {
```

Comment:
This function should have a `name` parameter that you use to generate the greeting in your code. If you call this function with any other name, it will not work as intended.